app/src/main/res/xml/remote_config_defaults.xml

```diff
@@ -228,4 +228,8 @@
         <key>v2_efgsTravellerDefault</key>
         <value>false</value>
     </entry>
+    <entry>
```

Comment:
Did you follow the steps for adding RC that David mention in other PR?
https://github.com/covid19cz/erouska-android/pull/501#discussion_r561945623

Reply:
removed, it will probably have to be done by @vitkolos